lib/src/model/robotoff_nutrient_extraction_annotation.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:openfoodfacts/src/model/nutrient_modifier.dart';
+import 'package:openfoodfacts/src/model/per_size.dart';
+import 'package:openfoodfacts/src/utils/unit_helper.dart';
+
+part 'robotoff_nutrient_extraction_annotation.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientAnnotationData {
+  @JsonKey(toJson: UnitHelper.unitToString, fromJson: UnitHelper.stringToUnit)
+  Unit? unit;
+  String valueWithModifer;
+
+  RobotoffNutrientAnnotationData({
+    this.unit,
+    required this.valueWithModifer,
+  });
+
+  get modifier => NutrientModifierExtension.fromValue(valueWithModifer);
```

Comment:
If possible I prefer explicit types rather than guessing them each time I read a code.
Besides, it's much safer - maintenance-wise.